src/main/java/mate/academy/util/HashUtil.java

```diff
@@ -0,0 +1,36 @@
+package mate.academy.util;
+
+import java.security.MessageDigest;
+import java.security.NoSuchAlgorithmException;
+import java.security.SecureRandom;
+
+public class HashUtil {
+    private static final String CRYPTO_ALGORITHM = "SHA-512";
+    private static final int SALT_ARRAY_SIZE = 16;
+
+    private HashUtil() {
+
+    }
+
+    public static byte[] getSalt() {
+        byte[] salt = new byte[SALT_ARRAY_SIZE];
+        new SecureRandom().nextBytes(salt);
```

Comment:
You are creating a new instance of SecureRandom for each use. This is unnecessary and can be inefficient as creating a new SecureRandom instance can be expensive. You should create a single instance and reuse it.

Reply:
Ok, thank you )